state-chain/pallets/cf-vaults/src/migrations/v3.rs

```diff
@@ -0,0 +1,82 @@
+use frame_support::traits::OnRuntimeUpgrade;
+use sp_std::{marker::PhantomData, prelude::*};
```

Comment:
```suggestion
use sp_std::{marker::PhantomData};

#[cfg(feature = "try-runtime")]
use sp_std::prelude::*;
```

same for threshold sig migration